send/scim_atlassian_mu.pm

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+SERVICE_NAME="scim_atlassian_mu"
+EXECSCRIPT="./scim-atlassian-connector.pl"
```

Comment:
We have a standard for the name convention of Perl execution scripts. In this case, it should be "scim_atlassian_mu.pl".